apps/content_types/content_types.py

```diff
@@ -180,6 +180,8 @@ def _update_template_fields(self, updates, original):
         Finds the templates that are referencing the given
         content profile an clears the disabled fields
         """
+        template_metadata_fields = ['language', 'usageterms']
```

Comment:
why is it just those 2 fields which are affected?

Reply:
Because these are the only fields of templates that don't depend on the schema.

Reply:
can you add some comment there regarding it? just to make it clear in the code.

Reply:
Sure, I'll.